src/app/app.component.ts

```diff
@@ -2,10 +2,11 @@ import { Component, OnInit, ViewChild, ViewEncapsulation } from '@angular/core';
 import { GridsterComponent, IGridsterDraggableOptions, IGridsterOptions } from 'angular2gridster';
 
 @Component({
-	selector: 'demo-root',
-	templateUrl: './app.component.html',
-	styleUrls: ['./app.component.css'],
-	encapsulation: ViewEncapsulation.None
+    selector: 'demo-root',
```

Comment:
For consistency, switch to indentation with tabs.

Reply:
@kyledickau This has been done.